types/ITestableWindow.d.ts

```diff
@@ -0,0 +1,6 @@
+export interface ITestableWindow {
```

Comment:
The types/ folder should be considered deprecated as we are trying to avoid global types. Could you move this interface somewhere else?